### PR TITLE
Update menu.lua

### PR DESCRIPTION
**Pull request by Zaps6000**
Fixing a nil error for a native not on redm

### CODE REVIEW

**Review by LukeWasTakenn — COMMENTED** [review]
Is there not a similar or equivalent native for RedM?

**Comment by Zaps6000** [comment]
> Is there not a similar or equivalent native for RedM?

If this is I couldnt find it I Just know that every time you open a menu it returns nil

**Comment by Zaps6000** [comment]
I looked again and am pretty sure there isn't one unless it isn't on the docs

File: resource/interface/client/menu.lua [hunk 1]
```diff
@@ -52,12 +52,15 @@ function lib.showMenu(id, startIndex)
     end
 
     if not openMenu then
+
```
Comment (LukeWasTakenn) [inline resource/interface/client/menu.lua:55]: Don't remove the space

File: resource/interface/client/menu.lua [hunk 2]
```diff
@@ -52,12 +52,15 @@ function lib.showMenu(id, startIndex)
...
         local control = cache.game == 'fivem' and 140 or 0xE30CD707
         CreateThread(function()
             while openMenu do
                 if openMenu.disableInput == nil or openMenu.disableInput then
                     DisablePlayerFiring(cache.playerId, true)
-                    HudWeaponWheelIgnoreSelection()
+                    if cache.game == 'fivem' then      
+                    HudWeaponWheelIgnoreSelection() -- Not a REDM native
```
Comment (LukeWasTakenn) [inline resource/interface/client/menu.lua:62]: This should be indented

File: resource/interface/client/menu.lua [hunk 3]
```diff
@@ -52,12 +52,15 @@ function lib.showMenu(id, startIndex)
...
+                        end
```
Comment (LukeWasTakenn) [inline resource/interface/client/menu.lua:63]: This should not be indented

**Review by LukeWasTakenn — CHANGES_REQUESTED** [review]
Fix the weird formatting issues and then it should be good to to be merged.

**Comment by Zaps6000** [comment]
> Fix the weird formatting issues and then it should be good to to be merged.

should be good

**Review by LukeWasTakenn — APPROVED** [review]

**Pull request merged**